src/js/components/FormField/FormField.js

```diff
@@ -334,7 +335,10 @@ const FormField = forwardRef(
       borderColor = (themeBorder && themeBorder.color) || 'border';
     }
 
-    const labelStyle = { ...formFieldTheme.label };
+    let labelStyle;
+    if (formKind) {
+      labelStyle = { ...formFieldTheme[formKind].label };
```

Comment:
Are there other style props that formKind could control besides just `label`?

Reply:
good point I could see us expanding on the `kind` for `border` or `roundness` I think we can brainstorm more how flexible we want `kind` but I agree on expanding

Reply:
I think it's fine to start with just what we need now, as long as we think the theme structure is reasonably future-proof, and add more later as tangible needs arise.